setup.py

```diff
@@ -32,11 +32,17 @@
         'License :: OSI Approved :: Zope Public License',
         'Framework :: Buildout',
         'Framework :: Plone',
-        'Framework :: Plone :: 5.1',
```

Comment:
At least the tests still pass on `5.1`. Is there a reason, we want to abandon it here without mentioning it in the changelog?

Reply:
No, I'll keep it for now.